src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -56,20 +56,84 @@ internal static class FeatureOnOffOptions
             nameof(FeatureOnOffOptions), nameof(AutoFormattingOnSemicolon), defaultValue: true,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.Auto Formatting On Semicolon"));
 
+        public static readonly PerLanguageOption<bool> IsCodeCleanupRulesConfigured = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(IsCodeCleanupRulesConfigured), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Is Code Cleanup Rules Configured"));
+
+        public static readonly PerLanguageOption<bool> RemoveUnusedUsings = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(RemoveUnusedUsings), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.Remove Unused Usings"));
+
+        public static readonly PerLanguageOption<bool> SortUsings = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(SortUsings), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.Sort Usings"));
+
+        public static readonly PerLanguageOption<bool> FixImplicitExplicitType = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixImplicitExplicitType), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Implicit Explicit Type"));
+
+        public static readonly PerLanguageOption<bool> FixThisQualification = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixThisQualification), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix This Qualification"));
+
+        public static readonly PerLanguageOption<bool> FixFrameworkTypes = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixFrameworkTypes), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Framework Types"));
```

Comment:
i don't know what "FixFrameworkTypes" means.

Reply:
I will change the name when we do the final string pass